package.json

```
@@ -21,7 +26,8 @@
         "dev-server": "encore dev-server",
         "dev": "encore dev",
         "watch": "encore dev --watch",
-        "build": "encore production --progress"
+        "build": "encore production --progress",
+        "eslint": "eslint assets/js --ext .js,.vue"
```

Maybe the alias command should be just "lint", this was a bit confusing, because you can run eslint by using `yarn eslint` aswell, and maybe I just want to use eslint, and run a linting process.
```suggestion
        "lint": "eslint assets/js --ext .js,.vue"
```